lib/storage/index.js

```diff
@@ -26,146 +26,144 @@ const validDataUrl = require('valid-data-url');
-            // but leave the rendition at its original path since worker.js still needs to read from it for metrics etc.
-            
-            // post processing does not apply for pipeline
-            const isPipelineRendition = false;
-            const newRendition = new Rendition(rendition.instructions, directories.out, rendition.index, isPipelineRendition);
     
-            console.log(`WORKER_TEST_MODE: copying ${rendition.path} to ${newRendition.path}`);
-            await fs.copy(rendition.path, newRendition.path);
+    static async putRendition(rendition, directories) {
+        // Note: validation has happened in validate.js before
+        if (process.env.WORKER_TEST_MODE) {
+            if (rendition.directory === directories.postprocessing) {
+                // copy the post processing rendition to the rendition path as expected by "aio asset-compute test-worker"
+                // but leave the rendition at its original path since worker.js still needs to read from it for metrics etc.
+                
+                // post processing does not apply for pipeline
+                const isPipelineRendition = false;
+                const newRendition = new Rendition(rendition.instructions, directories.out, rendition.index, isPipelineRendition);
+        
+                console.log(`WORKER_TEST_MODE: copying ${rendition.path} to ${newRendition.path}`);
+                await fs.copy(rendition.path, newRendition.path);
+        
+                rendition = newRendition;
+            }
     
-            rendition = newRendition;
-        }
-
-        // asset-compute-cli command run-worker wants file named as originally requested through "name"
-        // however, computing the metadata wants the current path, so we need to make a copy not just rename
-        if (rendition.instructions.name) {
-            const newPath = path.join(rendition.directory, rendition.instructions.name);
-
-            console.log(`WORKER_TEST_MODE: copying ${rendition.path} to ${newPath}`);
-            await fs.copyFile(rendition.path, newPath);
+            // asset-compute-cli command run-worker wants file named as originally requested through "name"
+            // however, computing the metadata wants the current path, so we need to make a copy not just rename
+            if (rendition.instructions.name) {
+                const newPath = path.join(rendition.directory, rendition.instructions.name);
+    
+                console.log(`WORKER_TEST_MODE: copying ${rendition.path} to ${newPath}`);
+                await fs.copyFile(rendition.path, newPath);
+            }
+    
+        } else if (!rendition.shouldEmbedInIOEvent()) {
+            await http.upload(rendition);
         }
-
-    } else if (!rendition.shouldEmbedInIOEvent()) {
-        await http.upload(rendition);
     }
 }
 
-module.exports = {
-    getSource,
-    putRendition,
-    getAsset
-};
\ No newline at end of file
+module.exports = Storage;
```

Comment:
nitpick, do we want to export it with `{}` so we can import it internally the same way we import it from external files:
internal
```js
const Storage = require('../../lib/storage'); // current
const {Storage} = require('../../lib/storage');
```

external
```js
const { Storage } = require('@adobe/asset-compute-pipeline');
```

Reply:
I don't have strong opinions about this, just wanted to bring it up in case others did

Reply:
Changed it to keep it consistent